pkg/operator/status.go

```diff
@@ -26,16 +26,19 @@ func (optr *Operator) syncAvailableStatus() error {
 	if co == nil {
 		return nil
 	}
-	now := metav1.Now()
+
+	optrVersion, _ := optr.vStore.Get("operator")
 	// set available
-	SetClusterOperatorStatusCondition(&co.Status.Conditions, configv1.ClusterOperatorStatusCondition{Type: configv1.OperatorAvailable, Status: configv1.ConditionTrue, LastTransitionTime: now})
+	cov1helpers.SetStatusCondition(&co.Status.Conditions, configv1.ClusterOperatorStatusCondition{
+		Type: configv1.OperatorAvailable, Status: configv1.ConditionTrue,
+		Message: fmt.Sprintf("Cluster is available at %s", optrVersion),
```

Comment:
s/Cluster/MCO/?  (or s/Cluster/Operator/, or...)

Reply:
MCO is reporting that its **view** (in terms of ownership) of the cluster is at that version.

Reply:
` - lastTransitionTime: 2019-01-29T20:53:23Z
    message: Cluster is available at 3.11.0-521-g377bde3b
    status: "True"
    type: Available
  - lastTransitionTime: 2019-01-30T20:56:32Z
    message: Running resync for 3.11.0-521-g377bde3b
    status: "True"
    type: Progressing
  - lastTransitionTime: 2019-01-30T06:21:34Z
    status: "False"
    type: Failing
`
@cgwalters it ends up reading this way in the yaml ^^

Reply:
OK, sounds fine to me!